src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage = () => {
+  const { tabId } = useParams();
+  const selectId = tabId;
+
+  return (
+    <div data-cy="TabsComponent">
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={classNames(
+                { 'is-active': tab.id === selectId },
+              )}
+              data-cy="Tab"
+            >
+              <Link to={`/tabs/${tab.id}`} data-cy="TabLink">
+                {tab.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find(t => t.id === selectId)
```

Comment:
You're using 'tabs.find' to check if the tab exists and then 'tabs.filter' to extract the tab. This can be optimized by just using 'tabs.find' and storing the result in a variable, which improves performance by reducing the number of iterations over the 'tabs' array.